assignments/boolean-birds/code.js

```diff
@@ -16,3 +16,34 @@
  *   - Globby Birds eat fish, mice, and nuts.
  *
  */
+const isFlobbyBird = (isRed,isSpotted) => isRed&&isSpotted
```

Comment:
Oh, also you should put spaces around operators and after commas. Like this:

```suggestion
const isFlobbyBird = (isRed, isSpotted) => isRed && isSpotted;
```